src/components/dialog/Dialog.vue

```diff
@@ -164,7 +165,8 @@ export default {
             prompt,
             isActive: false,
             validationMessage: '',
-            isCompositing: false
+            isCompositing: false,
+            Loading: false
```

Comment:
@zhanghengxin @wesdevpro 
I am wondering if the name `Loading` is relevant, because the consequence of `confirm` is not necessarily "loading". How about `isConfirming` or `isProcessing`?

Reply:
Whether it is in definition or development, isConfirming, isProcessing are unrelated to Loading. @kikuomax 

1、The original intent of Loading is to express the loading status of the confirm button. It is defined as a state machine and follows the old way of exposing the state machine change methods: startLoading, stopLoading.

2、There is a change to the line of code for isCompositing in the git record, which only added a comma.

3、Whether the Loading state changes is decided by the user, the user enables it through StartLoading. The default state of Loading is false.

I'm not sure if my replies have answered your question. Please let me know if you need any clarification or have additional questions.

Reply:
@zhanghengxin Thanks for clarification.

"Loading" makes sense in terms of the internal state of the confirm button. But how about if we want to, I think we should, disable the cancel button while "Loading" is on?

And how about "StartLoading" and "StopLoading" which are exposed to the user? It won't make sense if what the user is doing is not "loading".

Reply:
1、To maintain naming consistency and follow the naming rule aligned with isActive, change Loading to isLoading.

2、Then when Loading, change the status of the cancel button to disabled? Is this possible?

Reply:
But, if the cancel button is disabled, there may be a case where - the user's loading result fails, should it close the dialog, or keep it in the dialog interface?

Reply:
In other words, if we give the user control over the Loading state, then the user will be able to stay on the dialog interface when the loading result fails. I'm unsure if this violates the original intent of designing the dialog component.

Reply:
Currently, even if there is a possibility of a second dialog, the component does not have the ability to change feedback content. Generally feedback is provided via the Toast component.

Therefore, I think I can define the Dialog component as designed for single use dialogs. Then I can simply handle it by disabling the cancel button when isLoading is true.

Reply:
> But, if the cancel button is disabled, there may be a case where - the user's loading result fails, should it close the dialog, or keep it in the dialog interface?

It is up to the user whether the dialog is closed on failure. I mentioned `cancelLoading` (or `cancelConfirm`) method in [another comment](https://github.com/buefy/buefy/pull/3897#discussion_r1353830769) which was intended to be called on failure if the user wants to keep the dialog opened. The method will clear the "Loading" state.

Reply:
done

Reply:
@kikuomax waiting your review 😄